### ISSUE TITLE
Can't setup dvm with zsh

### ISSUE THREAD

**Issue by tonio-ramirez** [howtowhale/dvm#144]
### What didn't work?
```
$ source /home/username/.dvm/dvm.sh
/usr/bin/cd: line 2: cd: -q: invalid option
cd: usage: cd [-L|[-P [-e]] [-@]] [dir]
```

### What should it have done?
It should have setup dvm. 

### Helpful Context
What version of dvm are you using?

Not sure, since I wasn't able to set it up... I just used the command-line install option today (2017-01-04), so i'm guessing 0.6.5 or 0.6.4.

What is your operating System?

Fedora 25

What is your shell (bash/powershell/cmd/fish)?

zsh

How did you install dvm (command-line,chocolatey,homebrew)?

command-line

Do you like kittens? :smile_cat:

Yes.

**Comment by carolynvs** [comment]
Hmm, I thought that the `cd -q` tweak in dvm.sh was put in to make zsh work. I don't ever use zsh, so I'll need to poke around a bit to see what's what...

Does your version of zsh have a `-q` option for `cd`? http://zsh.sourceforge.net/Doc/Release/Shell-Builtin-Commands.html

**Comment by tonio-ramirez** [comment]
It does.  I managed to make it work by removing the "command" right before the call to "cd".  That is, changing:
```
export DVM_HELPER="$(command cd $DVM_CD_FLAGS "$(dirname "${DVM_SCRIPT_SOURCE:-$0}")" > /dev/null && command pwd)/dvm-helper/dvm-helper"
```
to
```
export DVM_HELPER="$(cd $DVM_CD_FLAGS "$(dirname "${DVM_SCRIPT_SOURCE:-$0}")" > /dev/null && command pwd)/dvm-helper/dvm-helper"
```

Note that calling `/usr/bin/cd -q .dvm` from the command line (zsh) also results in the error.

**Comment by carolynvs** [comment]
Thanks for the sleuthing! Looks like that was introduced by https://github.com/getcarina/dvm/pull/130/files. I now have a Fedora VM with zsh setup, so I think I can repro and work on a fix now.

**Comment by carloseduardosx** [comment]
Hello @carolynvs, I have fixed this problem on the #178 PR.
